Gruntfile.js

```diff
@@ -136,6 +136,10 @@ module.exports = function( grunt ) {
 				"test/cli/fixtures/only/module.js",
 				"test/cli/fixtures/only/module-flat.js",
 
+				"test/es2018/async-functions.js",
+				"test/es2018/rejects.js",
+				"test/es2018/throws.js"
+
 				// FIXME: These tests use an ugly hack that re-opens
```

Comment:
(Unrelated to this patch; This is probalby not true anymore as of https://github.com/qunitjs/qunit/pull/1629. I'll make a note at https://github.com/qunitjs/qunit/issues/1511 to revisit this.)